gui/datumaro_gui/components/multiple/streamlit_app.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (C) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+
+import os
+
+import streamlit as st
+import streamlit_antd_components as sac
+from datumaro_gui.utils.dataset.data_loader import DataRepo, MultipleDatasetHelper
+from datumaro_gui.utils.dataset.state import (
+    format_selector,
+    get_download_folder_path,
+    multiple_file_selector,
+    multiple_state_keys,
+    reset_state,
+    save_dataset,
+)
+from datumaro_gui.utils.drawing.css import box_style, custom_css
+from datumaro_gui.utils.readme import github_pypi_desc
+from streamlit import session_state as state
+
+from . import tabs
+
+
+def main():
+    print("__multiple__.main is called")
```

Comment:
I think print codes are for debugging. How about using logger for this?

Reply:
Fix this [4bc52ea](https://github.com/openvinotoolkit/datumaro/pull/1279/commits/4bc52eaec15dafe601fb7f7605907135e9f08c5f)